### PR TITLE
e2e tests : enable keycloak and quay test cases

### PR DESCRIPTION
**Pull request by subhashkhileri**
## Description

Enable keycloak and quay test cases and add/update the dynamic config required for plugin test cases.
Update the pipeline to run plugin test cases (keycloak and quay)

Please explain the changes you made here.

## Which issue(s) does this PR fix
https://issues.redhat.com/browse/RHIDP-859

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: .ibm/pipelines/value_files/values_showcase.yaml [hunk 1]
```diff
@@ -42,60 +42,57 @@ global:
...
-                        if:
-                          allOf:
-                            - isGithubPullRequestsAvailable
         - package: ./dynamic-plugins/dist/backstage-plugin-github-actions
           disabled: false
+        - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-quay
+          disabled: false
+        - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-keycloak-backend-dynamic
+          disabled: false
+          pluginConfig:
```
Comment (davidfestal) [inline .ibm/pipelines/value_files/values_showcase.yaml:53]: Why do you add the `pluginConfig` field here ? Do you want to explicitly replace the default config provided in [the included `dynamic-plugins.default.yaml` file](https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml#L536) ?

File: .ibm/pipelines/value_files/values_showcase.yaml [hunk 2]
```diff
@@ -42,60 +42,57 @@ global:
...
-                  mountPoints:
-                    - mountPoint: entity.page.ci/cards
-                      importName: EntityGithubActionsContent
-                      config:
-                        layout:
-                          gridColumn: "1 / -1"
-                        if:
-                          allOf:
-                            - isGithubActionsAvailable
+            kubernetes:
```
Comment (davidfestal) [inline .ibm/pipelines/value_files/values_showcase.yaml:66]: Why do you add the `pluginConfig` field here ? Do you want to explicitly replace the default config provided in [the included `dynamic-plugins.default.yaml` file](https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml#L174) ?

Comment (subhashkhileri) [reply .ibm/pipelines/value_files/values_showcase.yaml:66]: @davidfestal there is a additional config for topology code decorators
```
  - group: 'org.eclipse.che'
    apiVersion: 'v2'
    plural: 'checlusters'
```

Comment (davidfestal) [reply .ibm/pipelines/value_files/values_showcase.yaml:56]: I see.

**Review by gustavolira — APPROVED** [review]

**Review by josephca — APPROVED** [review]
/lgtm
/approve

**Review by gustavolira — APPROVED** [review]

**Comment by josephca** [comment]
@davidfestal can you review this pr? thanks!

**Review by kadel — APPROVED** [review]

**Pull request merged**